src/pages/resultsView/enrichments/MiniBoxPlot.tsx

```diff
@@ -163,8 +163,8 @@ export default class MiniBoxPlot extends React.Component<IMiniBoxPlotProps, {}>
                                 </VictoryChart>
                             </div>
                             {this.tooltipModel &&
-                                <Popover positionLeft={this.tooltipModel.x + 22} 
-                                    positionTop={this.tooltipModel.y - 22} className="cbioTooltip"
+                                <Popover positionLeft={this.tooltipModel.x + 16} 
```

Comment:
seems like we should strive to separate this kind of tweak from the logical change above into two commits.